skfuzzy/control/tests/test_controlsystem.py

```diff
@@ -53,7 +53,7 @@ def test_tipping_problem():
     tip_sim2.compute()
 
     assert tip_sim.output == tip_sim2.output
-    tst.assert_allclose(tip_sim.output['tip'], 20.244508118433625)
+    tst.assert_allclose(tip_sim.output['tip'], 19.847607362)
```

Comment:
Thank you! Therefore, I will change the assert to:
`tst.assert_allclose(tip_sim.output['tip'], 19.8578, atol=1e-2)`

This way it will reflect the exact value and the error associated.